src/components/GalleryLightBox/GalleryLightBox.tsx

```diff
@@ -199,9 +199,11 @@ export const GalleryLightBox = ({
                     )}
                     <div className="cc-gallery-lightbox__download">
                       <a
-                        href={slide.mediaSources.gallery_full_hi}
-                        download
                         className="cc-gallery-lightbox__download-link u-color-inherit"
+                        download
```

Comment:
I think download only works with same origin .. not sure what would happen in this case, re. headless?

Reply:
yeah you're right it doesn't work in our case but adding it in hopes we can weave some magic in future :)

Reply:
target blank added to open image in a new tab